src/Fluidity/Data/FluidityRepository`T.cs

```diff
@@ -20,7 +20,7 @@ namespace Fluidity.Data
     /// <typeparam name="TEntity">The type of the entity.</typeparam>
     /// <typeparam name="TId">The type of the identifier.</typeparam>
     /// <seealso cref="Fluidity.Data.IFluidityRepository" />
-    public abstract class FluidityRepository<TEntity, TId> : IFluidityRepository
+    public abstract class FluidityRepository<TEntity, TId> : IFluidityRepository, IDisposable
```

Comment:
Is IDisposable necesarry here as IFluidityRepository implement IDisposable anyway?

Reply:
It's not. It was only the `ObsoleteAttribute` on the interface that caused me to add it here, for fear that if the interface was removed without IDisposable being moved over to the abstract class. It sounds like this is unnecessary duplication though so I'll remove it. 2 secs.

Reply:
Yea, IFluidityRepository whilst marked as obsolete, won't be removed. It's purely to stop it being used directly.